pkg/kubernetes/deploy.js

```diff
@@ -0,0 +1,454 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+define([
+    "jquery",
+    "base1/cockpit",
+    "translated!base1/po",
+    "kubernetes/client",
+    "base1/mustache"
+], function($, cockpit,po,kubernetes,Mustache) {
+    "use strict";
+
+    cockpit.locale(po);
```

Comment:
The main cluster.html page should load the po. We don't need to do this here. I know I got this wrong in some other places too. Sorry bout that.